pkg/volume/azure_dd/azure_common_windows.go

```diff
@@ -103,8 +103,12 @@ func formatIfNotFormatted(disk string, fstype string, exec mount.Exec) {
 		return
 	}
 
+	if len(fstype) == 0 {
+		// Use 'NTFS' as the default
+		fstype = "NTFS"
+	}
 	cmd := fmt.Sprintf("Get-Disk -Number %s | Where partitionstyle -eq 'raw' | Initialize-Disk -PartitionStyle MBR -PassThru", disk)
-	cmd += " | New-Partition -AssignDriveLetter -UseMaximumSize | Format-Volume -FileSystem NTFS -Confirm:$false"
+	cmd += fmt.Sprintf(" | New-Partition -AssignDriveLetter -UseMaximumSize | Format-Volume -FileSystem %s -Confirm:$false", fstype)
```

Comment:
Which filesystems are supported now on Windows? Should we validate the type as well?

Reply:
@feiskyer There are NFTS, FAT32, REFS, etc. I would like to let user decide fstype, then in the future, if a new fstype is supported on Windows, we don't need to do any code change, same logic to Linux, it does not do any validation either.